Jenkinsfile

```diff
@@ -1115,6 +1116,8 @@ pipeline {
                     environment {
                         DOCKER_BUILDKIT        = '0'
                         DOCKER_DUT_DEBUG       = '1'
+                        DOCKER_DUT_HYPERV      = '1'
+                        DOCKER_STORAGE_OPTS    = 'size=40G'
```

Comment:
I guess that these settings should be under win-2022 section?

Reply:
Oh, wow, yes, you are right. Either has been a bad rebase at some point, or I may have been testing at the time if the "size" issue also reproduced on RS5 in Hyper-V